### PR TITLE
rpm2cpio: Install this tool to extract bzImage from rpm

### PR DESCRIPTION
**Pull request by bjzhjing**
Signed-off-by: Cathy Zhang <cathy.zhang@intel.com>

### CODE REVIEW

File: Dockerfile.x86_64 [hunk 1]
```diff
@@ -8,6 +8,9 @@ RUN apt-get update
 
 RUN apt-get -y install gcc
 
+# Install to extract bzImage
```
Comment (andreeaflorescu) [inline Dockerfile.x86_64]: This needs to be added to Dockerfile.aarch64 as well.

Comment (andreeaflorescu) [reply Dockerfile.x86_64]: nit: Install tool?

File: Dockerfile.x86_64 [hunk 2]
```diff
@@ -8,6 +8,9 @@ RUN apt-get update
 
 RUN apt-get -y install gcc
 
+# Install to extract bzImage 
+RUN apt-get -y rpm2cpio
```
Comment (andreeaflorescu) [inline Dockerfile.x86_64]: Tried to build the container on an x86_64 system:
```
Step 6/19 : RUN apt-get -y rpm2cpio
 ---> Running in 0500ec2e50e6
E: Invalid operation rpm2cpio
```

This line should be: ```RUN apt-get install -y rpm2cpio```

**Comment by rbradford** [comment]
@bjzhjing wouldn't it just be easier to use a .deb and extract it with `dpkg -x` ?

**Comment by andreeaflorescu** [comment]
@rbradford I think dpkg is already available in the container so we wouldn't need to update it. If it is possible to get the bzimage that way I think we should go on the path you suggested.

**Comment by rbradford** [comment]
@bjzhjing here is the kernel from debian stable: http://ftp.debian.org/debian/pool/main/l/linux/linux-image-4.9.0-9-amd64_4.9.168-1_amd64.deb won't be deleted from the archive for a couple of years at least.

**Comment by bjzhjing** [comment]
@andreeaflorescu @rbradford update post-checkout with dpdg-deb, it works as expected. I will close this PR.

**Pull request closed without merging**